.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: go
+
+go:
+  - 1.7.x
```

Comment:
I think we need to limit to 1.8+ at least because of gomacro's use of plugins.

Reply:
ah, right.

Reply:
gophernotes now states as prerequisite "Go 1.9+ required", and I also updated gomacro to require the same (see https://github.com/gopherdata/gophernotes/pull/104)